.NET/Microsoft.Recognizers.DataTypes.DataDrivenTests/Microsoft.Recognizers.DataTypes.DataDrivenTests.csproj

```diff
@@ -1,19 +1,87 @@
-<Project Sdk="Microsoft.NET.Sdk">
-
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="..\packages\MSTest.TestAdapter.1.2.0\build\net45\MSTest.TestAdapter.props" Condition="Exists('..\packages\MSTest.TestAdapter.1.2.0\build\net45\MSTest.TestAdapter.props')" />
```

Comment:
Any reason it points to TestAdapter for net45, but the target frameowork is net462 below?
Just curious.